### ISSUE TITLE
[iOS][Accessibility] [Input.Time/Date with a validation failure does not announce label and error after the submit]

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#4980]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* iOS

# Details
A new issue is found during the smoke test.
Input.Time/Date with a validation failure do not announce label and error messages after the submit
